act-rules-format.md

```diff
@@ -206,6 +218,84 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
+
+  <aside class=example>
+    <header>Example: a rule that tests if a focusable element has no keyboard trap via standard navigation</header>
+    <blockquote>This rule was designed to test for a specific solution that can be used to meet Success Criterion 2.1.2: No Keyboard Trap. The rule does not test for other possible solutions, such as the use of non-standard navigation, so its failed outcomes cannot determine that the accessibility requirement is <dfn>not satisfied</dfn>. This rule’s mapping:
+<ul>
+<li>Secondary Requirement: Success Criterion 2.1.2: No Keyboard Trap</li>
+  <ul><li>Background Section: This rule tests for one solution that can meet Success Criterion 2.1.2. SC 2.1.2 is mapped as a Secondary requirement because other solutions (such as use of non-standard navigation to exit a keyboard trap) can be used to meet this SC that are not tested by this rule.</li></ul>
+</ul>
+</blockquote>
+  </aside>
+
+**Scenario 3**: the rule is stricter than a requirement
+
+A rule was designed to test an accessibility requirement and there exists a less strict accessibility requirement. In this scenario, the rule’s passed outcomes can determine that the less strict requirement is satisfied, and the rule’s failed outcomes may not determine that the less strict requirement is not satisfied. It is possible that the accessibility requirement may be satisfied when the rule’s outcome is failed. The less strict accessibility requirement is a Secondary requirement.</li>
+
+<aside class=example>
+<header>Example: a rule that tests Enhanced Contrast</header>
+<blockquote>This rule was designed to test Success Criterion 1.4.6 Contrast (Enhanced) (Level AAA). A less strict requirement, Success Criterion 1.4.3 Contrast Minimum (AA), will be satisfied when the rule outcomes are passed, and may be satisfied when the rule outcomes are failed. This rule’s mapping:
+<ul>
+<li>Conformance Requirement: Success Criterion 1.4.6 Contrast (Enhanced)</li>
+<li>Secondary Requirement: Success Criterion 1.4.3 Contrast (Minimum)</li>
+<ul><li>Background Section: Success Criterion 1.4.3 is mapped as a Secondary requirement because the SC may be <dfn>satisfied</dfn>when the rule’s outcomes are `failed`.</li></ul>
+</aside>
+
+
+**Scenario 4**: the rule may impact the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement
+
+A rule was designed to test an accessibility requirement and under certain conditions, other accessibility requirements apply. In this scenario, the other accessibility requirements are sometimes, but not always, satisfied or not satisfied because they are not always applicable in the rule. These other accessibility requirements are a Secondary requirement.
```

Comment:
```suggestion
A rule was designed to test an accessibility requirement and under certain conditions, other accessibility requirements apply. In this scenario, the other accessibility requirements are only sometimes evaluated they are not always applicable in the rule. These other accessibility requirements are a Secondary requirement.
```
I found the second sentence here a bit confusing. I think this says the same thing but simplify the langauge a bit?

As a second question, does "not always applicable in the rule" == "there exists a passed or failed test case for which the requirement is applicable" (wording could be better). In my opinion, I think this should be the case, if it isn't then it means we are missing a test case. I think making "not always applicable in the rule" more concrete by referencing test cases makes it more measurable (e.g., applicable to some but not  all passed and failed examples == secondary requirement under scenario 4) and makes it more understandable.